shared/src/utils/is-development.ts

```diff
@@ -0,0 +1,10 @@
+// This environment is set at the backend and (manually) on the frontend, therefore we can use it in here
+declare global {
+    const process: {
+        env: {
+            NODE_ENV: 'development' | 'production';
```

Comment:
This type is incorrect. We use `migration` when running migrations. Also, there could be other `NODE_ENV`s that we are currently not using (common ones are e.g. `testing` or `staging`). So I'd rather use `string` here.